src/js/player.js

```diff
@@ -1328,6 +1328,16 @@ class Player extends Component {
     return this.tech_;
   }
 
+  /**
+   * Return a Video.js version. Follows [semantic versioning](https://semver.org/).
+   *
+   * @return {string}
+   *          Video.js version
+   */
+  version() {
+    return version;
```

Comment:
I think rather than returning just the string, we should return an object with `videojs` a property for this version.

This way, we can extend this to add plugin and tech versions in here too, similar to https://github.com/videojs/http-streaming/blob/main/src/videojs-http-streaming.js#L1173-L1180

Reply:
changed https://github.com/videojs/video.js/pull/8543/commits/3b10e8c42e795d7944cb5d5b89fc20d6a512ac5b